canvasapi/canvas.py

```diff
@@ -1184,6 +1184,32 @@ def get_user_participants(self, appointment_group, **kwargs):
             _kwargs=combine_kwargs(**kwargs),
         )
 
+    def graphql(self, query, variables):
+        """
+        Makes a GraphQL formatted requeset to Canvas
+
+        :calls: `POST /api/graphql \
+        <https://canvas.instructure.com/doc/api/file.graphql.html>`_
+
+        :param query: The GraphQL query to execute
+        :type query: str
+
+        :param variables: Variable values as required by the supplied query
+        :type variables: dict
+
+        :rtype: dict
+        """
+        response = self.__requester.request(
+            "POST",
+            "graphql",
+            _apiv="",
```

Comment:
Yeah, I didn't see this before. I mean it seems a little hacky but I also haven't seen anything other than /api/v1 and this /api for GraphQL. Instructure says all new functionality is going to get added to graphQL. I'd figure if they did significantly change something they'd go v2 (as that's the point of v1) which is why I added these changed but as you say this doesn't look required to accomplish this! I noticed the `_url `when working on another recent fix.

I can revert these changes and see what it looks like, I feel like this should be fine.

Reply:
Ok, this seems close but the problem I'm getting now is `__requester.base_url `gets set to be "/api/v1" in the Canvas `__init__ `. So a little more manipulation might be needed, or we'll have to store the original URL?

Reply:
I changed a little around here, appending the "/api/v1" in the requester instead of passing it into Canvas, then preserving the original URL. Let me know what you think about this.